lib/trento/application/integration/discovery/payloads/host_discovery_payload.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Trento.Integration.Discovery.HostDiscoveryPayload do
```

Comment:
In the checks code we used `DTO` instead of `Payload` to refer to this kind of messages.
Just as a comment

Reply:
@fabriziosestito @nelsonkopliku what do you think about this? Shall I rename this to DTO?